hadoop-yarn-project/hadoop-yarn/hadoop-yarn-client/src/main/java/org/apache/hadoop/yarn/client/cli/RMAdminCLI.java

```diff
@@ -1039,6 +1114,54 @@ protected String getUsageString() {
     return "Usage: rmadmin";
   }
 
+  /**
+   * Parse subClusterId.
+   * This method will only parse subClusterId when Yarn Federation mode is enabled.
+   *
+   * @param cliParser CommandLine.
+   * @return subClusterId.
+   */
+  protected String parseSubClusterId(CommandLine cliParser) {
+    // If YARN Federation mode is not enabled, return empty.
+    if (!isYarnFederationEnabled(getConf())) {
+      return StringUtils.EMPTY;
+    }
+
+    String subClusterId = cliParser.getOptionValue(OPTION_SUBCLUSTERID);
+    if(StringUtils.isBlank(subClusterId)) {
```

Comment:
Space missing.

Reply:
Thank you very much for your help in reviewing the code! I will fix it.